pkg/k8s/apis/cilium.io/v2/clrp_types.go

```diff
@@ -217,8 +217,13 @@ func (pInfo *PortInfo) SanitizePortInfo(checkNamedPort bool) (uint16, string, lb
 	}
 	// Sanitize name
 	if checkNamedPort {
+		if pInfo.Name == "" {
+			return pInt, pName, protocol, fmt.Errorf("port %s in the local "+
+				"redirect policy spec must be named as there are multiple ports", pInfo.Port)
+
+		}
 		if !iana.IsSvcName(pInfo.Name) {
-			return pInt, pName, protocol, fmt.Errorf("valid port name is not present")
+			return pInt, pName, protocol, fmt.Errorf("port name %s isn't valid", pInfo.Name)
```

Comment:
It reads better, but it would be ideal if we had a pointer to provide to tell users what are the valid names. Not sure if that's easy?

Reply:
Oh, there was a race with Daniel's review, I'm closing this item.